### PR TITLE
add s02e02 normal themes

### PR DESCRIPTION
**Pull request by IsaackMwenda**
Thematic analysis: https://docs.google.com/spreadsheets/d/1bWqhcVDH7AI3FFgVAz5TgrRw3AmQTR1jp4SAHoVZRno/edit#gid=0

### CODE REVIEW

File: code_schemes/dadaab_s02e02_reasons.json [hunk 1]
```diff
@@ -3,6 +3,62 @@
...
+      "CodeType": "Normal",
+      "DisplayText": "marry them off",
+      "NumericValue": 5,
+      "StringValue": "marry_them_off",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-52781d10",
+      "CodeType": "Normal",
+      "DisplayText": "advice them to stop",
```
Comment (as2388) [inline code_schemes/dadaab_s02e02_reasons.json]:
```suggestion
      "DisplayText": "advise them to stop",
```

File: code_schemes/kakuma_s02e02_reasons.json [hunk 2]
```diff
@@ -3,6 +3,63 @@
...
+    },
+    {
+      "CodeID": "code-917d66ff",
+      "CodeType": "Normal",
+      "DisplayText": "marrying girls early destroys their future",
+      "NumericValue": 5,
+      "StringValue": "marrying_girls_early_destroys_their_future",
+      "VisibleInCoda": true
+    },
+
```
Comment (as2388) [inline code_schemes/kakuma_s02e02_reasons.json:46]: I think you're missing one here, "marry her off", which is in the thematic doc (note also that this differs from "marry them off" in Dadaab, not sure if that's intentional)

**Review by as2388 — APPROVED** [review]

File: code_schemes/kakuma_s02e02_reasons.json (hunk 2 above)
Comment (IsaackMwenda) [reply code_schemes/kakuma_s02e02_reasons.json:46]: updated to marry her off in Dadaab for consistency.

File: code_schemes/dadaab_s02e02_reasons.json (hunk 1 above)
Comment (IsaackMwenda) [reply code_schemes/dadaab_s02e02_reasons.json]: updated thanks.

**Pull request merged**